rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.allocation/10_basic.yml

```diff
@@ -295,7 +295,7 @@
           ( [-\w.]+                \s+
             [-\w.]+                \s+
             [-\w.]+                \s+
-            [\w]+
+            [-\w.]+
```

Comment:
I tried finding out if something changed in the computation of that field - i.e. if it's a bug that we're returning a non-zero value - but wasn't able to identify anything.